examples/train_step_example.py

```diff
@@ -52,19 +52,13 @@
     criterion = (nn.CrossEntropyLoss())  # All classification problems we need Cross entropy loss
     early_stopper = utils.EarlyStopping(patience=7, verbose=True, path=SAVE_PATH)
 
```

Comment:
You should probably not even touch this file.
This is example which was created to show `train_step`. This is going to be part of documentation and kills the purpose of file.
Please use a seperate `dev.py` file and do not commit it / push to github.

Reply:
ohh sorry actually I did it for testing purpose  but anyways why to use 2 different functions if engine.fit can be used for train and valid set

Reply:
There is deeper reason why `train_step` and `val_step` are present.
The reason being, it gives really powerful control and allows a lot of in between operations. Have a look at examples.